dat/missions/neutral/pirbounty_dead.lua

```diff
@@ -381,6 +393,7 @@ function spawn_pirate( param )
          pir_jump_hook = hook.pilot( target_ship, "jump", "pilot_jump" )
          pir_land_hook = hook.pilot( target_ship, "land", "pilot_jump" )
 
+	 misn.appendLog("Target pirate ship sighted")
```

Comment:
Assuming the log is supposed to be a useful reference for important past events, this is just cluttering it up with unnecessary information.